addons/aws/1.0.1/kubeadm-cluster-config-v1beta2.yml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: kubeadm.k8s.io/v1beta2
+kind: ClusterConfiguration
+metadata:
+  name: kubeadm-cluster-configuration
+clusterName: $KUBERNETES_CLUSTER_NAME
+networking:
+  podSubnet: $POD_CIDR
+  serviceSubnet: $SERVICE_CIDR
+apiServer:
+  extraArgs:
+    cloud-provider: aws
+    enable-admission-plugins: DefaultStorageClass
+controllerManager:
+  extraArgs:
+    cloud-provider: aws
+    configure-cloud-routes: '"true"'
+    address: 0.0.0.0
```

Comment:
can you add a comment justifying why this is necessary? otherwise remove it

Reply:
[removed](https://replicated.slack.com/archives/C02TC9ZN519/p1652123682611509?thread_ts=1652117140.465169&cid=C02TC9ZN519)